script/Arcueid/Day 10/10_00_ARC10_5C.txt

```diff
@@ -265,7 +265,7 @@ Arcueid doesn't seem to feel like sitting down for whatever reason,
 -- Page 521, Offset 25613. Mods: @n.
 -- 時間だけがすぎていく
 // 'Only time passes' as in nothing else happens, I assume
-Everything else is still save for the hands of the clock
+Everything else is still, save for the hands of the clock
```

Comment:
Don't think I like this comma but I would drop the `else`

Reply:
personally I'd keep the comma and drop the "else"
> Everything is still, save for the hands of the clock

is how i'd naturally pause it if reading for dramatic effect